.gitlab-ci.yml

```diff
@@ -305,7 +305,7 @@ test-runtime-benchmarks:
       - $DEPLOY_TAG
   script:
     - cd bin/node/cli
-    - BUILD_DUMMY_WASM_BINARY=1 time cargo check --verbose --features runtime-benchmarks
+    - BUILD_DUMMY_WASM_BINARY=1 time cargo test --verbose --features runtime-benchmarks
```

Comment:
Does it make sense to have my own CI which runs all tests again with this feature flag?

Or should I simply add my feature flag to the main testing?

Reply:
i guess it's probably fine to just add it to the main testing.

Reply:
ahh, I cant do it actually because the main test works off of the virtual manifest, which does not accept feature flags :/